### ISSUE TITLE
[v5] bug: Segment should not emit onChange event until pointer end

### ISSUE THREAD

**Issue by liamdebeasi** [ionic-team/ionic#20257]
# Bug Report

**Ionic version:**

[x] **5.0.0-beta.5**

**Current behavior:**

When dragging the new segment design via a gesture, the onChange events are emitted whenever the segment indicator moves from one button to another. This is inconsistent with the native iOS segment spec where the onChange event is emitted only after the user's pointer has been removed from the screen.

**Expected behavior:**

I would expect that the onChange event is emitted after the user's pointer has been removed from the screen.

**Steps to reproduce:**

1. Open https://codepen.io/liamdebeasi/pen/KKwbvJY
2. Click/tap and drag the segment indicator.
3. Notice that the "Num times changed" count increases every time the segment indicator moves.

**Other information:**

This has performance implications as the onChange event may be tied to hiding/showing elements in a user's application. For example, if changing the segment button toggles the visibility of a div, dragging the indicator dozens of times is going to cause dozens of change detections to run in Angular apps. If the onChange event is only fired when the user removes their pointer from the screen, then change detection is only going to run once.

**Comment by danieljameswilliams** [comment]
I am still experiencing something related to this issue in 5.0.1

If you scroll while touching a segment, it selects it.
But it also throws this error in console:

```
[Intervention] Ignored attempt to cancel a touchend event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
```

**Comment by liamdebeasi** [comment]
@danieljameswilliams  Please open a new issue
